### PR TITLE
Fix: [AEA-4205] - redact step function logs before sending to splunk

### PR DESCRIPTION
**Pull request by anthony-nhs**
## Summary

- Routine Change

### Details

- redact input, output, parameters from step function logs before sending to splunk

### CODE REVIEW

File: packages/splunkProcessor/src/splunkProcessor.js [hunk 1]
```diff
@@ -132,7 +132,7 @@ function transformStepFunctionsLogEvent(logEvent) {
     eventMessage["X-Amzn-Trace-Id"] = normalizedHeaders["x-amzn-trace-id"]
     eventMessage["x-correlation-id"] = normalizedHeaders["x-correlation-id"]
     eventMessage["x-request-id"] = normalizedHeaders["x-request-id"]
-  } catch (_) {
```
Comment (wildjames) [inline packages/splunkProcessor/src/splunkProcessor.js:135]: Is it me, or is this weirdly indented? Does this repo have an automatic linter? Also, if you'll allow me to nitpick a little, I'd space the try/catch and the triple-if on line 144 by one line, to distinguish them visually (extremely minor point)

File: packages/splunkProcessor/tests/transformLogEvent.stepfunctions.test.js [hunk 2]
```diff
@@ -38,15 +38,7 @@ describe("transformLogEvent tests for stepFunctions log groups", () => {
...
-              headers: {
-                "apigw-request-id": "ac9ddfe0-030a-4bcd-b63b-d4d1a32381a0",
-                "X-Amzn-Trace-Id": "Root=1-6613feca-23ef52160d128597232a4c97",
-                "x-correlation-id": "b9156b99-13cc-4719-b832-73905e07e760",
-                "x-request-id": "a22cc83d-b40b-43a1-b5f7-71944cd498bc"
-              }
-            })
+            input: "redacted"
           },
           "apigw-request-id": "ac9ddfe0-030a-4bcd-b63b-d4d1a32381a0",
```
Comment (wildjames) [inline packages/splunkProcessor/tests/transformLogEvent.stepfunctions.test.js:43]: Just checking that it's okay to have the IDs present here?

Comment (anthony-nhs) [reply packages/splunkProcessor/tests/transformLogEvent.stepfunctions.test.js:43]: ids need to present so we can correlate the logs together with ones from other systems.
Or if you mean having the actual ids in this file, then its fine as its just a file for unit tests

File: packages/splunkProcessor/src/splunkProcessor.js (hunk 1 above)
Comment (anthony-nhs) [reply packages/splunkProcessor/src/splunkProcessor.js]: its weirdly formatted. Have corrected and pushed a change to enable format on save on this folder

**Review by wildjames — APPROVED** [review]

**Pull request merged**